### ISSUE TITLE
Potential Fire Hazard

### ISSUE THREAD

**Issue by SenorFusion** [Roastero/Roastero#22]
Per Sweet Marias (and other places) roasting coffee beans can easily catch fire at or above bean temperatures of 495F-505F degrees.

http://www.sweetmarias.com/library/content/using-sight-determine-degree-roast (see Step 16)

I realize that the temperatures used on Roastero and FreshRoast are the air temperature, but in #4 increasing air temperature max to 550F could easily bring bean temperatures to 500+

-Perhaps we should talk to the manufacturer about the temperatures the machine was designed for?

-Possibly include a big warning label on the Graph and in Roast Settings when selecting temps above 500?

**Comment by lodge93** [comment]
We increased the upper limit to 550 as a user mentioned they roast up past 500. The way our software works is it alternates between cool, low, medium, and high in order to attempt to reach the target temperature. If we set the temperature at 550, it would be no different than a user setting the roaster on high and letting it heat up. So at first look, it would appear that we are giving the roaster cadaverous capabilities. In reality we are still bound by the "high" setting of the roaster.
